### PR TITLE
Features/main view design

### PR DESCRIPTION
**Pull request by gilemos**
Hey guys! I changes a little bit of the design of the first screen in this branch. See if you liked it

Video Walkthrough

There is also an animation in the line when it just loads

Video Walkthrough

### CODE REVIEW

**Review by aliu917 — APPROVED** [review]
The new look is really nice, I like it! Also, the loading bar is so cool!

File: Travel Scheduler/View Controllers/FirstScreenViewController.m [hunk 1]
```diff
@@ -87,8 +90,53 @@ - (void)viewDidLoad
...
+    [self setUpImage];
+    
+    
+    for(NSArray *family in UIFont.familyNames) {
+        for(NSString *name in [UIFont fontNamesForFamilyName:family]) {
+            NSLog(@"%@", name);
+        }
+    }
+}
+
```
Comment (aliu917) [inline Travel Scheduler/View Controllers/FirstScreenViewController.m]: Is this testing stuff?

File: Travel Scheduler/View Controllers/FirstScreenViewController.m [hunk 2]
```diff
@@ -87,8 +90,53 @@ - (void)viewDidLoad
...
     self.autocompleteTableView.delegate = self;
     self.autocompleteTableView.dataSource = self;
+    [self makeAnimatedLine];
+    [self setUpImage];
+    
+    
+    for(NSArray *family in UIFont.familyNames) {
+        for(NSString *name in [UIFont fontNamesForFamilyName:family]) {
+            NSLog(@"%@", name);
+        }
```
Comment (aliu917) [inline Travel Scheduler/View Controllers/FirstScreenViewController.m]: Is this testing stuff?

File: Travel Scheduler/View Controllers/FirstScreenViewController.h [hunk 3]
```diff
@@ -27,7 +27,7 @@ NS_ASSUME_NONNULL_BEGIN
 @property (strong, nonatomic) NSString *userSpecifiedPlaceToVisit;
 @property (strong, nonatomic) NSDate *userSpecifiedStartDate;
 @property (strong, nonatomic) NSDate *userSpecifiedEndDate;
-@property (strong, nonatomic) UILabel *headerLabel;
+//@property (strong, nonatomic) UILabel *headerLabel;
```
Comment (frankboamps) [inline Travel Scheduler/View Controllers/FirstScreenViewController.h]: I this commented out or will still be in use for the future. Consider removing it if not in use.

**Review by frankboamps — APPROVED** [review]
On the whole you did an awesome job with the First Page UI and changing the text font. Really liked the UI additions. One issue is to consider deleting commented code and ensure that your comments are consistent. Saw that comment for AttractionCollectionCell.h lines 74 and 75 were inconsistent with your usual comment style where you have comments before code.

File: Travel Scheduler/View Controllers/FirstScreenViewController.m (hunk 2 above)
Comment (gilemos) [reply Travel Scheduler/View Controllers/FirstScreenViewController.m]: lol yeah, i was trying to find the correct name for the font I wanted. Gonna erase that, thanks for pointing out!

**Comment by gilemos** [comment]
Great comments guys, thank you!! I have already fixed what you guys pointed out

**Pull request merged**